pkg/kv/kvserver/replica_range_lease.go

```diff
@@ -1216,6 +1221,11 @@ func (r *Replica) redirectOnOrAcquireLeaseForRequest(
 		// against this in checkRequestTimeRLocked). So instead of assuming
 		// anything, we iterate and check again.
 		pErr = func() (pErr *roachpb.Error) {
+			// NB: the slow request detection here is not particularly useful since
+			// it guards the actual lease proposal, which is a proposal and which
+			// thus already has a "slow message" trigger attached to it.
+			//
+			// TODO(during review): decide whether to remove this right now.
```

Comment:
One argument for keeping this is that the slow proposal logging will only log the oldest request, and so it can mask the lease acquisition. We may want to add conditional slow request logging for significant system proposals such as lease acquisitions.

Given how delicate the lease acquisition stuff is, we may want to leave this in for another major release cycle, and tear it out once we're more comfortable with the new breakers. That will also retain this logging here if we either end up disabling breakers by default, or users disable it explicitly.